src/mango/test/user_docs.py

```diff
@@ -58,9 +60,18 @@ def setup_users(db, **kwargs):
     db.save_docs(copy.deepcopy(USERS_DOCS))
 
 
-def setup(db, index_type="view", **kwargs):
+def setup(db, index_type="view", partitioned=False, **kwargs):
     db.recreate()
-    db.save_docs(copy.deepcopy(DOCS))
+    p = str(partitioned).lower()
+    docs = copy.deepcopy(DOCS)
+
+    if partitioned:
+        for index, doc in zip(itertools.count(), docs):
```

Comment:
would `enumerate(docs)` work here?

Reply:
Yes, thanks for the suggestion.  That is even shorter.